myway-api/src/main/java/com/hmel/myway/controllers/DestinationController.java

```diff
@@ -74,7 +74,7 @@
 		logger.info("Get all destination ");
 		List<Destination> res = new ArrayList<>();
 		try {
-			res = iDestinationService.findAll();
+			res = iDestinationService.findAll(0,Integer.MAX_VALUE);
```

Comment:
if name of methods 'findAll' it is must search all records, not 50 or 100. In our entity primary key is long so max value can be Long.MAX_VALUE but it is realy very big value(and we don't need it now), so it will be better use now Integer.MAX_VALUE. It is normal practice for search all records for entity to use max value from type of primary key. If you select some type for key you add also some rules with identified from type. Need to speak about this. It is not a problem to use some limit, We can declare it in service without problems